### PR TITLE
Cribl Stream Initial Pull Request

### PR DESCRIPTION
**Pull request by amiracle**
## Pull Request Description

This pull request introduces a new Cribl solution for Azure Sentinel, which includes the following changes:

### Cribl Steam DataConnector
- Added a new folder named "Cribl-Stream" in the "DataConnector" directory
- Created a new file named "CriblStream.json" inside the "DataConnector" folder

### Cribl Logo
- Added Cribl-Logo.svg to "Logos" directory
## Reasons for Change(s):
- Cribl Stream DataConnectors being added into the Microsoft Azure Sentinel repository.   
- Version 1.0.0 Added.
- Tested and validated DataConnector file with preview function in Sentinel.
-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by amiracle** [comment]
> @microsoft-github-policy-service agree [company=“Cribl"]
> @microsoft-github-policy-service agree
> @microsoft-github-policy-service agree company=“Cribl"





Kam Amir, Dir. Technical Alliances 
(240) 466-1385 | ***@***.*** ***@***.***> | calendar <https://calendly.com/kamiloamir/mwkam>
Cribl : Data Engine for IT And Security

**Comment by amiracle** [comment]
@microsoft-github-policy-service agree company="Cribl"

File: Solutions/Intel471/Package/mainTemplate.json
Comment (v-prasadboke) [inline Solutions/Intel471/Package/mainTemplate.json:1]: Hello @amiracle, Please discard the changes to this Solution.

**Comment by v-prasadboke** [comment]
Hello @amiracle, You have added this Data connector in standlalone folder. If this data connector is a part of Solution then please create a new folder with Solution name in Solution's folder. 

Also add input file located in Data folder and solution metadata to the Solution folder which are required to package the solution. 

You can refer this Solution for more clarification. 
https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Agari

**Comment by v-prasadboke** [comment]
Also can you provide me write access to this branch so that I can commit required changes. 

You can do this by inviting me as collaborator

**Comment by amiracle** [comment]
> Also can you provide me write access to this branch so that I can commit required changes.
> 
> You can do this by inviting me as collaborator

I've granted you write access to the repo.

**Comment by v-prasadboke** [comment]
> Hello @amiracle, You have added this Data connector in standlalone folder. If this data connector is a part of Solution then please create a new folder with Solution name in Solution's folder.
> 
> Also add input file located in Data folder and solution metadata to the Solution folder which are required to package the solution.
> 
> You can refer this Solution for more clarification. https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Agari

Hello @amiracle thanks for providing write access. 
Wanted to check if you have any updates on the above requested files

**Comment by v-prasadboke** [comment]
Hello @amiracle, You might be busy with something. 
Just wanted to hop in to check if you have any updates on the above required commits.

**Comment by amiracle** [comment]
> Hello @amiracle, You might be busy with something. Just wanted to hop in to check if you have any updates on the above required commits.
Let me start this PR again with a proper components added to the repo.

**Pull request closed without merging**